clients/js/test/freezeV2.test.ts

```
@@ -0,0 +1,666 @@
+  // And a 9th minted NFT owned by leafOwnerA.
+  const leafOwner = generateSigner(umi);
+  const { metadata, leaf, leafIndex } = await mintV2(umi, {
+    merkleTree,
+    leafOwner: leafOwner.publicKey,
+    leafIndex: 8,
+  });
+
+  // And given we mock the RPC client to return the following asset and proof.
+  let merkleTreeAccount = await fetchMerkleTree(umi, merkleTree);
+  const [assetId] = findLeafAssetIdPda(umi, { merkleTree, leafIndex });
+  let rpcAsset = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'rpcAsset' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)